frame/transaction-payment/asset-tx-payment/src/lib.rs

```diff
@@ -97,6 +97,7 @@ pub(crate) type ChargeAssetLiquidityOf<T> =
 #[derive(Encode, Decode, DefaultNoBound, TypeInfo)]
 pub enum InitialPayment<T: Config> {
 	/// No initial fee was payed.
+	#[default]
```

Comment:
Why did you change this?

Reply:
the `#[default]` attribute is now required:
```
error: no default declared, make a variant default by placing `#[default]` above it
  --> frame/transaction-payment/asset-tx-payment/src/lib.rs:98:10
   |
98 | pub enum InitialPayment<T: Config> {
   |          ^^^^^^^^^^^^^^
```

Reply:
Ahh it was just deriving default using the first variant...